backend/API/API/HostedServices/IsSolvedService.cs

```diff
@@ -35,7 +31,7 @@ protected async override Task ExecuteAsync(CancellationToken stoppingToken)
         {
             while (!stoppingToken.IsCancellationRequested)
             {
-                QueueService.Models.ReceiveData receiveData = await Task.Run(() => consumer.Receive(2500));
+                ReceiveData receiveData = await Task.Run(() => consumer.Receive(2500));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4635982)